telegram/_writeaccessallowed.py

```diff
@@ -26,28 +26,54 @@
 class WriteAccessAllowed(TelegramObject):
```

Comment:
Do we want to add id_attrs ot this class? I guess `web_app_name` is a must have. `from_request` and `from_attachment_menu` can make sense, but I wouldn't insist on them. If we don't use them for equality comparison, then this should go into a standalone PR …

Reply:
Wdym "not use them for equality comparison", thats the point of id_attrs, no?

Reply:
No, that's what I meant: I don't insist on adding `from_*` to `_id_attrs` and if we don't, then the change (adding only `web_app_name`) should go into a different pr

Reply:
yea I would prefer having just `web_app_name` in a new pr

Reply:
#3909